k8s-install/config.yaml

```diff
@@ -0,0 +1,36 @@
+# This ConfigMap can be used to configure a self-hosted Canal installation.
+# See `canal.yaml` for an example of a Canal deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: canal-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
+  etcd_endpoints: "http://127.0.0.1:2379"
+
+  # The interface used by flannel for host <-> host communication.
+  flannel_iface: "eth1"
```

Comment:
https://github.com/coreos/flannel#key-command-line-options

I think it's just `--iface` and `--public-ip`.  I didn't expose the latter, since it doesn't make sense for a DaemonSet install.